packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -305,7 +305,7 @@ test('price drop', async t => {
   const {
     vaultFactory: { vaultFactory, vfPublic },
     priceAuthority,
-    reserveKit: { reserveCreatorFacet },
+    reserveKit: { creatorFacet: reserveCreatorFacet },
```

Comment:
was this a latent bug?

Reply:
this was to change less in the test file when renaming `reserveCreatorFacet` in `reserveKit`